src/frontends/tensorflow_common/src/op/inv.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_op_table.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/divide.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+OutputVector translate_inv_op(const NodeContext& node) {
+    default_op_checks(node, 1, {"Inv"});
+    auto x = node.get_input(0);
+
+    // prepare auxiliary one constants of the same type as the inputs
+    auto one = make_shared<v0::Constant>(element::i32, Shape{}, 1);
```

Comment:
please create `one` constant of the same type as `x` using `create_same_type_const`. Otherwise, we will have type conflict